### ISSUE TITLE
Rename and follow spring boot versions

### ISSUE THREAD

**Issue by paulvi** [avaje-pom/tile-runnable-jar#1]
Rename and follow spring boot versions,

so if it add Spring Boot loader 1.3.3.RELASE, so should be the version of this time. IMHO

**Comment by rbygrave** [comment]
Yes I understand. I have a "runnable war" shade/parent pom style that I want to alternatively do as a spring boot loader / runnable jar style.  So part of me thinks I should revisit the names at that point.

I do prefer just having major.minor version numbers for all composite poms / tiles / parent poms / maven plugins ... so 1.3.3.RELEASE isn't going to conform with that.

**Comment by paulvi** [comment]
Would 1.3.3 feel right ?
